modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -64,6 +64,14 @@ type Handler struct {
 	// on this property long-term; check release notes.
 	PathPrefix string `json:"path_prefix,omitempty"`
 
+	// Whether to store using FileIO rather than MemoryMap
+	// Deafult: false
+	NoMemoryMap bool `json:"no_memory_map,omitempty`
+
+	// Whether to use BadgerV2 storage
+	// Deafult: false
+	UseBadgerV2 bool `json:"use_badgerv2,omitempty`
```

Comment:
Please see the golangci-lint errors in the diff https://github.com/caddyserver/caddy/pull/3849/files#diff-47200afac47ea8df3c0a5e4b0c7b00872d3d40287b0327ae1cccfc90161e4606R73

Reply:
Also I think `use_badger_v2` is nicer.

Reply:
Yea. I think it's nicer too. I used `bagerv2` because that's what the value is called, but I guess that's not really important and definitely makes it harder to read.